src/safeposix/syscalls/net_calls.rs

```diff
@@ -938,10 +1193,17 @@ impl Cage {
             };
         }
 
+        //Setup the socket handle as connected, insert the remote address of the
+        // connection, and reset the errno in case it is set to EINPROGRESS
         sockhandle.state = ConnState::CONNECTED;
         sockhandle.remoteaddr = Some(remoteaddr.clone());
         sockhandle.errno = 0;
-        // set the rawfd for select
+        // Set the rawfd for select_syscall as we cannot implement the select
```

Comment:
I think this is somewhat misphrased, its not that we cannot implement select its just that we use libc to do so. also "kernel"